src/fiber/execution_context/parallel/scheduler.cr

```diff
@@ -29,13 +29,23 @@ module Fiber::ExecutionContext
       @spinning = false
       @waiting = false
       @parked = false
+      @shutdown = Shutdown::NO
 
       protected def initialize(@execution_context, @name)
         @global_queue = @execution_context.global_queue
         @runnables = Runnables(256).new(@global_queue)
         @event_loop = @execution_context.event_loop
       end
 
+      # :nodoc:
+      enum Shutdown
+        NO  = 0
+        NOW
+      end
```

Comment:
`Shutdown::NO` is weird. If someone has a better suggestion...

Reply:
Do we even need an enum type for such an internal detail? Why not make it a boolean instead?

Reply:
We're planning to tell an execution context to shutdown and the behavior will be different to shutdown _now_ (drain your local queue and stop immediately) and will instead be _when nothing left to do_ (stop instead of parking) for example.